.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 20.18.2
```

Comment:
Is is possible to leave a comment in this file to add what tool this file is for?

Reply:
Will do in a subsequent PR!